src/components/forms/RangeInput/RangeInput.tsx

```diff
@@ -17,47 +19,38 @@ type RangeInputProps = {
 export const RangeInput = ({
   className,
   inputRef,
+  textPreposition,
+  textUnit,
   ...inputProps
 }: RangeInputProps & JSX.IntrinsicElements['input']): React.ReactElement => {
   const classes = classnames('usa-range', className)
   // input range defaults to min = 0, max = 100, step = 1, and value = (max/2) if not specified.
   const defaultMin = 0
   const defaultMax = 100
-  const { min, max, defaultValue } = inputProps
+  const defaultVal = Math.round(defaultMax / 2)
+  const defaultUnit = ''
+  const defaultPreposition = 'of'
+  const { min, max } = inputProps
```

Comment:
```suggestion
  const { min, max, defaultValue } = inputProps
```

Reply:
I also wonder if we should parse the `value` prop from `inputProps` and try to use that in case someone is passing it in 🤔 

That would look like 

```suggestion
  const { min, max, defaultValue, value: valueProp } = inputProps
```

Reply:
Just realized, `defaultValue` passed in still gets applied to the input triggering the console error. If we pull out `remainingProps` and then spread them below, its all good

```suggestion
  const {
    min,
    max,
    defaultValue,
    value: valueProp,
    ...remainingInputProps
  } = inputProps
```